packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Expressions/InvokeInstancePropertyExpression.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Generator.CSharp.Statements;
+
+namespace Microsoft.Generator.CSharp.Expressions
+{
+    public sealed record InvokeInstancePropertyExpression(ValueExpression InstanceReference, string PropertyName) : ValueExpression
```

Comment:
we shouldn't need this since ValueExpression already has a `Property` method which returns MemberExpression